Week2/homework/index.js

```diff
@@ -0,0 +1,125 @@
+const express = require('express');
+const mysql = require('mysql');
+const bodyParser = require('body-parser');
+
+const app = express();
+
+app.use(express.json());
+
+const connection = mysql.createConnection({
+  host: 'localhost',
+  user: 'root',
+  password: 'password',
+  database: 'new_world',
+});
+
+connection.connect();
+
+const data = require('./cities.json');
+const { countries } = data;
+const { cities } = data;
+
+function getCapitalCity(id, req, res) {
+  connection.query(`select * from cities where id = "${id}"`, (error, results, fields) => {
```

Comment:
Avoid injecting user data like that :"${id}". Use prepared statement instead ( where id = ? ).

Reply:
👍